src/components/AreaChart/AreaChart.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { createStyles, makeStyles } from '@material-ui/core/styles';
 import { Box, Typography } from '@material-ui/core';
 import Chart from 'react-apexcharts';
-import moment from 'moment';
+import dayjs from 'dayjs';
```

Comment:
Why? I understand dayJs is suppose to be a minifed version of moment but it doesn't have the same credibility of the original package.

Reply:
I think we don't need to use both of moment and dayjs because dayjs has all functions which moment has. Tested on my side and was able to get all values we got using moment with dayjs.